### PR TITLE
Homepge

### PR DESCRIPTION
**Pull request by tamana-Balkhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+hello world is the first project of microverse.
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
please make your readme more descriptive please follow the [requirements](https://github.com/microverseinc/readme-template/blob/master/README.md)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <p>
+      hello world is the first project of microvese that should be created at
+      second day of microverse program wiht all profesional dacumentation.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate cum
+      sapiente maiores possimus inventore, facilis repellendus dicta fugiat
+      incidunt sed est commodi quam ratione libero laborum modi dignissimos.
+      Officia, porro!
+    </p>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `optional`
Great job going an extra mile and adding more initiative however this was not part of the requirement but I suggest you stick to the requirement for professionalism.

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @tamana-Balkhi 👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Solid pull request scroll
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

- [ ] please make your readme more descriptive please follow the [requirements](https://github.com/microverseinc/readme-template/blob/master/README.md)

- [ ] gitignore is only supposed to have node_modules and test.md see [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @tamana-Balkhi , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**